calyx/src/passes/well_formed.rs

```diff
@@ -185,6 +185,34 @@ impl Visitor for WellFormed {
             }
         }
 
+        // If the component is combinational, make sure all cells are also combinational
+        // and there are no group or comb group definitions
```

Comment:
We also need to make sure that the control program is empty

Reply:
Added a check for this, but there should be no way to create this based on the changes to the parser.